tests/unit/parallel_test.py

```diff
@@ -67,6 +69,31 @@ def f(obj):
     assert errors == {}
 
 
+def test_parallel_execute_with_global_limit():
```

Comment:
Make sure to use a `@mock.patch.dict(os.environ)` decorator to avoid bleeding test values into the user's environment and prevent unexpected side-effects with other tests.

Reply:
This is no longer required as the code for the test no longer sets the environment variable the way it used to